src/obb_rules/auto_deploys/common.cljc

```diff
@@ -51,7 +51,7 @@
   "Builds a deploy action command"
   [player row idx [u q]]
   (let [coordinate (translator/coordinate player [(+ idx 1) row])]
-    [:deploy (int q) (unit/unit-name u) coordinate]))
+    [:deploy (int q) (keyword (unit/unit-name u)) coordinate]))
```

Comment:
Yes, this is a problem I faced when I tried do use the engine from json. Did some martelanços and eventually creates a [simplifyer ns](https://github.com/orionsbelt-battlegrounds/obb-rules/blob/master/src/obb_rules/simplifier.cljc) to handle all that. Now I only use keywords, but there is still remnants of this on the code.